Vagrantfile

```diff
@@ -174,28 +170,23 @@ Vagrant.configure(2) do |config|
   config.hostmanager.ignore_private_ip = false
 
   # configure vagrant-vbguest plugin
-  if Vagrant.has_plugin?('vagrant-vbguest')
```

Comment:
hmm ok... in this case what do you think about these options

1. `&&` 
  ```ruby
  # configure vagrant-vbguest plugin
  Vagrant.has_plugin?('vagrant-vbguest') && config.vbguest.auto_update = true
  ```

2. verbose comments ( well actually I made this 'cause of this cop https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/if_unless_modifier.rb#L17 )

  ```ruby
  # Vagrant Plugin Configuration: vagrant-vbguest
    if Vagrant.has_plugin?('vagrant-vbguest')
      # enable auto update guest additions
      config.vbguest.auto_update = true
    end
  ```
  its just keeping the rule not doing single line blocks.

there is also an third option deactivating the cop. But it adds a useless line for everybody not using rubocop 😄 

i also can change it back to what you did so you can accept the rest.

just let me know